src/main/java/com/chat/yourway/service/AuthenticationService.java

```diff
@@ -91,11 +90,13 @@ public AuthResponseDto register(RegisterRequestDto request, HttpServletRequest h
     @Transactional
     public AuthResponseDto authenticate(AuthRequestDto request) {
         log.info("Started authenticate contact email: {}", request.getEmail());
-        authManager.authenticate(new UsernamePasswordAuthenticationToken(request.getEmail(), request.getPassword()));
 
         var contact = contactRepository.findByEmail(request.getEmail())
                 .orElseThrow();
 
+        if (!passwordEncoder.matches(request.getPassword(), contact.getPassword()))
```

Comment:
Its generally bad practice to use if without "{}" brackets, could you please add them?